File: backend-go/internal/modules/system/entity/sys_dict_item.go
package entity

import (
	"drip-admin/backend-go/internal/common"
	"time"
)

type SysDictItem struct {
	ID         common.Int64String `gorm:"column:id;primaryKey" json:"id"`
	DictTypeID common.Int64String `gorm:"column:dict_type_id" json:"dictTypeId"`
	Label      string             `gorm:"column:label" json:"label"`
	Value      string             `gorm:"column:value" json:"value"`
	IsDefault  int                `gorm:"column:is_default" json:"isDefault"`
	Sort       int                `gorm:"column:sort" json:"sort"`
	Status     int                `gorm:"column:status" json:"status"`
	Builtin    int                `gorm:"column:builtin" json:"builtin"`
	CreatedAt  *time.Time         `gorm:"column:created_at" json:"createdAt"`
	UpdatedAt  *time.Time         `gorm:"column:updated_at" json:"updatedAt"`
}

func (SysDictItem) TableName() string { return "sys_dict_item" }

File: backend-python/app/modules/system/entity/sys_dict_item.py
from sqlalchemy import BigInteger,Integer,String
from sqlalchemy.orm import Mapped,mapped_column
from app.modules.system.entity.base import Base,TimestampMixin
class SysDictItem(TimestampMixin,Base):
 __tablename__="sys_dict_item"
 id:Mapped[int]=mapped_column(BigInteger,primary_key=True)
 dict_type_id:Mapped[int]=mapped_column(BigInteger)
 label:Mapped[str]=mapped_column(String(64))
 value:Mapped[str]=mapped_column(String(64))
 is_default:Mapped[int]=mapped_column(Integer)
 sort:Mapped[int]=mapped_column(Integer)
 status:Mapped[int]=mapped_column(Integer)
 builtin:Mapped[int]=mapped_column(Integer)
